src/static-build/pages/index/index.tsx

```diff
@@ -27,7 +28,25 @@ const Index: FunctionalComponent<Props> = () => (
       <title>Squoosh</title>
       <meta
         name="description"
-        content="Compress and compare images with different codecs, right in your browser"
+        content="Squoosh is the ultimate image optimizer that allows you to compress and compare images with different codecs in your browser."
+      />
+      <meta
+        name="keywords"
+        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, shrink image size"
```

Comment:
Don't yo forget the AVIF image format 😜

```suggestion
        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, jpg to avif, shrink image size"
```